Isotopes.Rmd

```diff
@@ -0,0 +1,86 @@
+---
+title: "Isotopes Fractionation"
+subtitle: "Isoptopes are isoDOPE"
+author: "Kevin Rozmiarek"
+date: "Last knitted `r format(Sys.Date(), '%d %b %Y')`"
+output:
+  html_document: 
+    df_print: paged
+    number_sections: yes
+    toc: yes
+    toc_float: true
+    toc_depth: 3
+    code_folding: show
+editor_options:
+  chunk_output_type: inline
+---
+
+# Learning Goals
+
+1. Understand the role of **fractionation factors** in separating isotopes in different reservoirs
+
+2. Understand the difference between **equilibrium isotope fractionation** and **kinetic isotopic fractionation**
+
+3. Be able to numerically demonstrate the separation of two isotopic reservoirs by evaporation. 
```

Comment:
It seems the students wouldn't necessarily know how to demonstrate what the other reservoir would look like. Keep learning goals more specific, in this case maybe "numerically demonstrate the isotopic evolution of a water reservoir as a consequence of evaporation"?